manual/docker/docker-compose/ce/11.0/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
   memcached:
     image: memcached:1.6.18
     container_name: seafile-memcached
-    entrypoint: memcached -m 256
+    command: memcached -m 256
```

Comment:
Actually the entrypoint is
```
$ cat /entrypoint.sh 
#!/bin/sh
set -e

# first arg is `-f` or `--some-option`
if [ "${1#-}" != "$1" ]; then
        set -- memcached "$@"
fi

exec "$@"
```

So this can be simplified as

```suggestion
    command: -m 256
```